src/libs/actions/Session.js

```diff
@@ -139,8 +139,9 @@ function signIn(password, exitTo, twoFactorAuthCode) {
                     setSuccessfulSignInData(createLoginResponse, exitTo);
 
                     // If we have an old login for some reason, we should delete it before storing the new details
-                    if (credentials.login) {
+                    if (credentials.login && credentials.password) {
                         API.DeleteLogin({
+                            authToken: authenticateResponse.authToken,
```

Comment:
This request was failing on every sign in before since the `authToken` is not set yet.